lib/graphql-error-handler.js

```diff
@@ -0,0 +1,17 @@
+let graphqlErrorHandler = null;
+
+if (process.env.NEW_RELIC_LICENSE_KEY) {
+  const newrelic = require('newrelic');
+  graphqlErrorHandler = error => {
+    newrelic.noticeError(error, { graphql: true });
```

Comment:
I decided to not add the error cloning right now, because I’m thinking that the tagging might not actually be that relevant in this case.